db/db.go

```diff
@@ -43,8 +43,8 @@ type KVStore interface {
 	Delete(string, []byte) error
 	// Commit commits a batch
 	Commit(KVStoreBatch) error
-	// CommitWithFillPercent commits a batch with specified bucket fill percent
-	CommitWithFillPercent(KVStoreBatch, float64) error
+	// SetBucketFillPercent sets specified fill percent for a bucket
+	SetBucketFillPercent(string, float64) error
```

Comment:
KVStore as an interface doesn't need to include this API. You can create a KVStoreWithBucketFillPercent as an extend of KVStore.

Reply:
done in latest push